crates/iota-json-rpc/src/authority_state.rs

```diff
@@ -64,7 +64,6 @@ pub trait StateRead: Send + Sync {
         &self,
         transactions: &[TransactionDigest],
         effects: &[TransactionDigest],
-        events: &[TransactionEventsDigest],
```

Comment:
I think you missed a rename here.
```
   transaction_keys: &[TransactionDigest],
   effects_keys: &[TransactionDigest],
```

Maybe we could do that rename even in other functions like `get_object_read` etc?

Reply:
fixed with https://github.com/iotaledger/iota/pull/5259/commits/6e96d70faba0427d46f04ccb8791d5b15f1a7dfd